addons/stock/migrations/13.0.1.1/end-migration.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 ForgeFlow <https://www.forgeflow.com>
+# Copyright 2020 Tecnativa - Pedro M. Baezaw
```

Comment:
```suggestion
# Copyright 2020 Tecnativa - Pedro M. Baeza
```
I was just admiring this work when I saw this.

Reply:
done